File: test/python/producer1.py
from confluent_kafka import Producer

print("in")
conf = {'bootstrap.servers': "broker-1:19092"}
producer = Producer(conf)

def delivery_report(err, msg):
    if err is not None:
        print('Delivery failed:', err)
    else:
        print('Message delivered to', msg.topic(), msg.partition())

producer.produce('camera-job', key='key', value='Hello, Kafka!', callback=delivery_report)
producer.flush()

File: test/node/producer1.js
const { Kafka, logLevel, CompressionTypes } = require("kafkajs");

async function main() {
  const kafka = new Kafka({
    clientId: "my-app",
    brokers: ["192.168.1.11:29092"],
    logLevel: logLevel.INFO,
  });

  const producer = kafka.producer({ allowAutoTopicCreation: true });
  await producer.connect();
  await producer.send({
    topic: "camera-job",
    compression: CompressionTypes.GZIP,
    messages: [
      { key: "key1", value: '{"msg": "hello world"}', },
      { key: "key2", value: '{"msg": "hey hey!"}', },
      { key: "key1", value: '{"msg": "hello world"}', },
      { key: "key2", value: '{"msg": "hey hey!"}', },
      { key: "key1", value: '{"msg": "hello world"}', },
      { key: "key2", value: '{"msg": "hey hey!"}', },
      { key: "key1", value: '{"msg": "hello world"}', },
      { key: "key2", value: '{"msg": "hey hey!"}', },
    ],
  });

  await producer.disconnect();
}
main();
